CMSIS/Core/Include/cmsis_compiler.h

```diff
@@ -53,6 +53,21 @@
  */
 #elif defined ( __ICCARM__ )
 
+  /* CMSIS compiler control architecture macros */
+  #if (__CORE__ == __ARM6M__) || (__CORE__ == __ARM6SM__)
```

Comment:
Hi @0xc0170. Are the __CORE__ issues solved in the meanwhile by using latest IAR patch level? Please update this PR to a bare minimum.